### ISSUE TITLE
No mention of snapd’s API

### ISSUE THREAD

**Issue by matthewpaulthomas** [canonical-docs/snappy-docs#373]
The snapd wiki has [a frequently-updated API reference](https://github.com/snapcore/snapd/wiki/REST-API) (usually [updated](https://github.com/snapcore/snapd/wiki/REST-API/_history) by @robert-ancell, but occasionally by @caldav).

Unfortunately, docs.snapcraft.io [seems to have no mention of the API at all](https://www.google.com/search?q=rest+api+site%3Adocs.snapcraft.io), making people much less likely to know that it exists.

**Comment by robert-ancell** [comment]
I recently found https://docs.ubuntu.com/core/en/reference/rest by accident - this seems to be copied across from the API reference I update and formatted a bit more nicely.

**Comment by matthewpaulthomas** [comment]
Yes, those look to be [manual imports](https://github.com/canonical-docs/ubuntu-core-docs/commits/master/docs/en/reference/rest.md) by @caldav.

If it makes sense for the API to be described both on snapcraft.io and on docs.ubuntu.com/core, perhaps a script should sync it to both places at once. (Alternatively, if the source moves to docs.snapcraft.io, a script could sync it from there to docs.ubuntu.com/core.)
